oomact_python/python/oomact/tools.py

```diff
@@ -0,0 +1,68 @@
+import sm
+from sm import quatInv, quatIdentity
+import tf.transformations
+
+try:
+    from termcolor import colored
+except exceptions.ImportError:
```

Comment:
To get it to work on mine I had to change this to "except ImportError:"

Reply:
Thanks for telling! That way it also works on trusty (oldest supported Ubuntu). I've committed the change.